Wikipedia/Code/ArticleViewController+LinkPreviewing.swift

```diff
@@ -93,7 +93,6 @@ extension ArticleViewController: ArticleContextMenuPresenting, WKUIDelegate {
     func webView(_ webView: WKWebView, contextMenuForElement elementInfo: WKContextMenuElementInfo, willCommitWithAnimator animator: UIContextMenuInteractionCommitAnimating) {
         guard
```

Comment:
I noticed the popping is a bit slow on iPad, like it waits a while for the preview to disappear, then it triggers, whereas iPhone seems to pop in place as soon as you tap the preview. But when I add this line:

```
animator.preferredCommitStyle = .pop
```

It seems to pop quickly, like iPhone. I'm not sure why, in fact the [documentation](https://developer.apple.com/documentation/uikit/uicontextmenuinteractioncommitstyle?language=objc) seems to suggest the opposite (dismiss = no animation, pop = animation). Anyways just thought I'd mention it as a consideration, but I'm not sure if this is the right thing to do in this case. At any rate it looks good for approval.